synapse/http/servlet.py

```diff
@@ -169,6 +168,37 @@ def _parse_string_value(
         return value
 
 
+def parse_list_from_args(
```

Comment:
I think this attempts to do the same thing as `parse_strings_from_args` -- note that has incorrect type hints though.

Reply:
> note that has incorrect type hints though.

See #10137

Reply:
I've switched to `parse_strings_from_args` and removed `parse_list_from_args`. Thanks for cleaning that up separately!